makefile

```diff
@@ -0,0 +1,20 @@
+up:
+	docker-compose up -d
+
+stop:
+	docker-compose stop
+
+php:
+	docker-compose exec php-fpm sh
+
+mysql:
+	docker-compose exec mysql sh
+
+mailhog:
```

Comment:
Did you had cases when you had to enter in this container?

Reply:
No. Looks like it is redundant, indeed.